tdvt/tdvt/config_gen/test_config.py

```diff
@@ -250,6 +250,35 @@ def build_config_name(prefix, dsname):
 def build_tds_name(prefix, dsname):
     return prefix + dsname + '.tds'
 
+class TabQueryPath(object):
+    sep = ';'
```

Comment:
Do we want to use `os.path.sep` here to account for different seperators on different os's?